integration-libs/cdc/jest.ts

```diff
@@ -0,0 +1,36 @@
+// uncomment when we switch the whole lib to jest
+/*
+Object.defineProperty(window, 'CSS', { value: null });
+Object.defineProperty(window, 'getComputedStyle', {
+  value: () => {
+    return {
+      display: 'none',
+      appearance: ['-webkit-appearance'],
+    };
+  },
+});
+
+Object.defineProperty(document, 'doctype', {
+  value: '<!DOCTYPE html>',
+});
+Object.defineProperty(document.body.style, 'transform', {
+  value: () => {
+    return {
+      enumerable: true,
+      configurable: true,
+    };
+  },
+});
+*/
```

Comment:
1. I wonder if/when we will transition all unit tests to jest. If we have plans to do it, let's create a proposal ticket for it and let's put the required steps (code snippets) there in the ticket instead of putting long dead code comments. Or preferably let's put into the ticket just a link to a guideline how to migrate to Jest (I guess you copy pasted those snippets from some guideline). 
1. Just curious: Why we added this comment specifically only for a few integration libs (e.g. cdc, cds, dp)? But not for other libs? 
3. btw. Those commands are opinionated workarounds, overwriting global window object's properties. I would like to avoid them at all, if possible. In the future, if/when we transition to Jest, I hope they won't be needed. 🤞

Reply:
1. I think such ticket already exists.
2. Actually I must admit, it was copy&paste between different locations. I could have thought of it better, sorry.
3. Agreed, all comments will be gone from all these types of files. 👍🏼

Reply:
ad 1. What do you think about removing those commented out code snippets out from the to the ticket description?

Reply:
1. Done 👍🏼  #15004